x-pack/plugins/uptime/server/graphql/pings/schema.gql.ts

```diff
@@ -185,7 +185,7 @@ export const pingsSchema = gql`
     "The timestamp of the ping's creation"
     timestamp: String!
     "Milliseconds from the timestamp to the current time"
-    millisFromNow: Int
+    millisFromNow: Int64
```

Comment:
See my comment below - I think it'd be better to use a String here, and avoid having to maintain another scalar at this point.